src/app.js

```diff
@@ -1,13 +1,55 @@
 const express = require ('express');
+const { Cat } = require ('./models');
```

Comment:
Code here is fine, though the comments arent really helping, they are just describing _how_ you are doing what you are and not _why_, which is a far more valuable thing to provide to whoever is trying to understand and work on your code. As it is, this is clear enough to understand without having to add the comments and so they could all be removed.